Failed to install using both from git and luarocks

Hi,
I'm using ec2 instance with el6 OS and trying to install tarantool-mysql using both guide (git & luarocks), but i got these; any help will be greatly appreciated. :)

[root@ tarantool-mysql]# cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo
-- The C compiler identification is GNU 4.8.3
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find TARANTOOL (missing: TARANTOOL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:315 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindTarantool.cmake:26 (find_package_handle_standard_args)
  CMakeLists.txt:15 (find_package)


-- Configuring incomplete, errors occurred!
See also "/usr/share/tarantool/tarantool-mysql/CMakeFiles/CMakeOutput.log".
[root@ tarantool-mysql]# luarocks install https://raw.githubusercontent.com/tarantool/mysql/master/mysql-scm-1.rockspec --local
Using https://raw.githubusercontent.com/tarantool/mysql/master/mysql-scm-1.rockspec... switching to 'build' mode
Cloning into 'mysql'...
remote: Counting objects: 31, done.
remote: Compressing objects: 100% (24/24), done.
remote: Total 31 (delta 1), reused 14 (delta 0), pack-reused 0
Receiving objects: 100% (31/31), 15.07 KiB | 7.53 MiB/s, done.
Resolving deltas: 100% (1/1), done.
-- The C compiler identification is GNU 4.8.3
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find TARANTOOL (missing: TARANTOOL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:315 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindTarantool.cmake:26 (find_package_handle_standard_args)
  CMakeLists.txt:15 (find_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/luarocks_mysql-scm-1-1886/mysql/CMakeFiles/CMakeOutput.log".

Error: Build error: Failed cmake.

No `https://raw.githubusercontent.com/tarantool/mysql/master/mysql-scm-1.rockspec ` anymore

Using `https://raw.githubusercontent.com/tarantool/mysql/master/rockspecs/mysql-scm-1.rockspec`:

```
% luarocks install --local https://raw.githubusercontent.com/tarantool/mysql/master/rockspecs/mysql-scm-1.rockspec

Cloning into 'mysql'...
remote: Enumerating objects: 33, done.
remote: Counting objects: 100% (33/33), done.
remote: Compressing objects: 100% (26/26), done.
remote: Total 33 (delta 2), reused 15 (delta 0), pack-reused 0
Receiving objects: 100% (33/33), 15.68 KiB | 7.84 MiB/s, done.
Resolving deltas: 100% (2/2), done.
-- The C compiler identification is GNU 7.3.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Found TARANTOOL: /usr/include (found version "1.10.2-119-g21f87c30d") 
-- Tarantool LUADIR is /home/d9k/.luarocks/lib/luarocks/rocks-5.1/mysql/scm-1/lua
-- Tarantool LIBDIR is /home/d9k/.luarocks/lib/luarocks/rocks-5.1/mysql/scm-1/lib
CMake Error at CMakeLists.txt:37 (add_subdirectory):
  The source directory

    /tmp/luarocks_mysql-scm-1-5098527/mysql/mariadb-connector-c

  does not contain a CMakeLists.txt file.


-- Configuring incomplete, errors occurred!
See also "/tmp/luarocks_mysql-scm-1-5098527/mysql/build.luarocks/CMakeFiles/CMakeOutput.log".

Error: Build error: Failed cmake.
```

--- a/README.md
+++ b/README.md
@@ -6,28 +6,47 @@
 
 ### Prerequisites
 
- * Tarantool 1.6.5+ with header files (tarantool && tarantool-dev packages)
- * MySQL 5.1 header files (libmysqlclient-dev package)
- * OpenSSL development package
+* Tarantool 1.6.5+ with header files (tarantool && tarantool-dev /
+  tarantool-devel packages).
+* MySQL 5.1 header files (libmysqlclient-dev package).
+* OpenSSL development package.
+
+If you prefer to install the connector using a system package manager you don't
+need to manually install dependencies.
 
 ### Installation
 
+#### Build from sources
+
 Clone repository and then build it using CMake:
 
-``` bash
+```sh
 git clone https://github.com/tarantool/mysql.git tarantool-mysql
 cd tarantool-mysql && cmake . -DCMAKE_BUILD_TYPE=RelWithDebInfo
 make
 make install
 ```
 
-You can also use LuaRocks:
+#### tarantoolctl rocks
 
-``` bash
-luarocks install https://raw.githubusercontent.com/tarantool/mysql/master/mysql-scm-1.rockspec --local
+You can also use tarantoolctl rocks:
+
+```sh
+tarantoolctl rocks install mysql
 ```
 
-See [tarantool/rocks][TarantoolRocks] for LuaRocks configuration details.
+#### Install a package
+
+[Enable tarantool repository][tarantool_download] and install tarantool-mysql
+package:
+
+```sh
+apt-get install tarantool-mysql # Debian or Ubuntu
+yum install tarantool-mysql     # CentOS
+dnf install tarantool-mysql     # Fedora
+```
+
+[tarantool_download]: https://www.tarantool.io/en/download/
 
 ### Usage
 

